plugins/modules/aci_fabric_node.py

```diff
@@ -46,6 +46,15 @@
     type: str
     aliases: [ role_name ]
     choices: [ leaf, spine, unspecified ]
+  node_type:
+    description:
+    - Type for the new Fabric Node Member.
+    type: str
+    choices: [ tier-2, virtual, unspecified ]
```

Comment:
Think we should use _ for the choice inputs instead of -. 
```suggestion
    choices: [ tier_2, virtual, remote, unspecified ]
```

Is there a reason to not have remote in this list?